website/content/en/preview/reference/metrics.md

```diff
@@ -50,6 +50,26 @@ Amount of time required for a replacement machine to become initialized.
 ### `karpenter_deprovisioning_replacement_machine_launch_failure_counter`
 The number of times that Karpenter failed to launch a replacement node for deprovisioning. Labeled by deprovisioner.
 
+## Disruption Metrics
+
+### `karpenter_disruption_actions_performed_total`
+Number of disruption methods performed. Labeled by disruption type.
```

Comment:
does this make more sense?
```suggestion
Number of disruption actions performed. Labeled by disruption method.
```

Reply:
Ooops. Yep, should probably change this in core, then

Reply:
Updated